packages/create-redwood-app/template/api/src/lib/prisma.ts

```diff
@@ -0,0 +1,91 @@
+import { PrismaClient, Prisma } from '@prisma/client'
+
+import { logger } from './logger'
+
+// To help you identify particularly slow parameter values,
+// For example, Heroku outputs the slowest queries (that take 2 seconds or more)
+const SLOW_QUERY_THRESHOLD = 2 * 1000 // 2 seconds
```

Comment:
Underscores can be numerical separators
```suggestion
const SLOW_QUERY_THRESHOLD = 2_000 // 2 seconds
```

Reply:
Done